challenges/4-emprestimo/javascript/saraaniceto/index.js

```diff
@@ -0,0 +1,5 @@
+const [emprestimo, juros, tempo] = process.argv[2].split(',')
+
+let valorFinal = +emprestimo * Math.pow((1 + (+juros / 100)), +tempo)
+
+console.log(valorFinal.toFixed(2))
```

Comment:
Quase lá! mas muito boa solução novamente, ao inves de fixar as casas decimais em 2, vc pode usar a função Math.round para aproximar

```suggestion
console.log(Math.round(valorFinal))
```

Embaixo vou deixar os erros:
![image](https://github.com/he4rt/he4rtoberfest-2023/assets/86631177/b7e1c1e3-74e3-4464-b93f-239046eb35ca)

Reply:
mas o round não vai arredondar pra inteiro? No enunciado falava pro output ser um ponto flutuante com duas casas decimais.

Reply:
Ah perdão! foi erro meu, confundi totalmente aqui com a ferramenta que usamos para testar

Esta perfeito <3